style.css

```diff
@@ -88,3 +112,143 @@ p {
   gap: 20px;
   align-items: center;
 }
+
+/* my resent works section */
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.about-text h2 {
+  font-size: 40px;
+  margin-bottom: 24px;
+}
+
+.recent-works .text h2 {
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  margin: 24px 26px;
+}
+
+.recent-works .work-cards {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.about-cards .card {
+  background-color: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin: 12px 0;
+  padding-top: 40px;
+  padding-bottom: 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.recent-works .work-cards .card {
+  margin: 24px 20px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.recent-works .work-cards .card .card-text {
+  background-color: #fff;
+  margin-top: 250px;
+  border-radius: 8px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.recent-works .work-cards .card .card-text h3 {
+  color: #3a4a42;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+.recent-works .work-cards .card .card-text .work-catagories {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+}
+
+.about-cards .card .about-catagories li {
+  background-color: #fff;
+  padding: 12px;
+  border-radius: 8px;
+  color: #36b37e;
+}
+
+.recent-works .work-cards .card .card-text .work-catagories li {
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+  font-size: 12px;
+}
+
+.recent-works .work-cards .project-one {
+  background-image: url(./icons/Img-Plaholder.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* aboutme section */
+.about-me {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, please ensure you stick to the design as much as possible. Please add the background images serving as decoration to the about me section.

### Your design
![Aboutme2](https://user-images.githubusercontent.com/63964646/164563258-2fc0d85a-9683-4799-8ea6-825f371f930a.PNG)

### Original design
![Aboutme](https://user-images.githubusercontent.com/63964646/164563282-a024bd9a-47e0-4baa-8c9c-0b87e477fea5.PNG)